src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -57,15 +57,15 @@ public Transaction BuildTransaction(TransactionBuildContext context)
         {
             this.InitializeTransactionBuilder(context);
 
-            if (context.Shuffle)
-                context.TransactionBuilder.Shuffle();
-
-            const int maxRetries = 3;
+            const int maxRetries = 5;
             int retryCount = 0;
 
             TransactionPolicyError[] errors = null;
             while (retryCount <= maxRetries)
             {
+                if (context.Shuffle)
```

Comment:
What does shuffle do internally? its not just setting a flag?

Reply:
It is a flag we usually set to true, there is a comment against it: "We shuffle transaction outputs by default as it's better for anonymity.". I have just added this change to make it in line with code hat was already merged into 3.0.6 branch.